### PR TITLE
Roulette tgui-next overhaul

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

image

I got lazy towards the end and didn't do everything I wanted to.
Oh well, it's done and works.

## Changelog
:cl:
add: tgui-next roulette
fix: mule bot interface not working properly
/:cl:

### CODE REVIEW

File: code/game/machinery/roulette_machine.dm [hunk 1]
```diff
@@ -90,19 +102,11 @@
...
-			var/amount = input(usr, "Bet amount between 10 and 500:") as num|null
-			if(amount)
-				chosen_bet_amount = CLAMP(amount, 10, 500)
 		if("ChangeBetType")
 			chosen_bet_type = params["type"]
 			. = TRUE
 	update_icon() // Not applicable to all objects.
 
-/obj/machinery/roulette/ui_base_html(html)
-	var/datum/asset/spritesheet/simple/assets = get_asset_datum(/datum/asset/spritesheet/simple/roulette)
```
Comment (AnturK) [inline code/game/machinery/roulette_machine.dm]: Remove the asset and the images.

Comment (actioninja) [reply code/game/machinery/roulette_machine.dm]: oh yeah, good catch

**Comment by kriskog** [comment]
Merge conflict

**Review by stylemistake — DISMISSED** [review]
Yeah, that's a pretty complex one, looks alright.

File: code/game/machinery/roulette_machine.dm [hunk 2]
```diff
@@ -275,8 +294,25 @@
...
+			//numbers in the first column
+			var/list/winners = list(1, 4, 7, 10, 13, 16, 19, 22, 25, 28, 31, 34)
+			return (rolled_number in winners)
+		if(ROULETTE_BET_2TO1_SECOND)
+			//numbers in the second column
+			var/list/winners = list(2, 5, 8, 11, 14, 17, 20, 23, 26, 29, 32, 35)
+			return (rolled_number in winners)
+		if(ROULETTE_BET_2TO1_THIRD)
+			//numbers in the third column
+			var/list/winners = list(3, 5, 8, 12, 15, 18, 21, 24, 27, 30, 33, 36)
```
Comment (kriskog) [inline code/game/machinery/roulette_machine.dm]: 3, 6, 9, 12?

**Pull request merged**